ddtrace/contrib/dbapi/__init__.py

```diff
@@ -50,11 +50,12 @@ def _trace_method(self, method, name, resource, extra_tags, *args, **kwargs):
             s.set_tags(pin.tags)
             s.set_tags(extra_tags)
 
-            # set analytics sample rate if enabled
-            s.set_tag(
-                ANALYTICS_SAMPLE_RATE_KEY,
-                config.dbapi2.get_analytics_sample_rate()
-            )
+            # set analytics sample rate if enabled but only for non-FetchTracedCursor
+            if type(self) is not FetchTracedCursor:
```

Comment:
We inherit from this class in other integrations, so you'd want, `if not isinstance(self, FetchTracedCursor):`